extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -1,10 +1,308 @@
+
+- Google hosted: _coming soon_
+
+[tip type="note"]
+Before setting up a backend, consider if the already existing backends satisfy your requirements.
+[/tip]
+
+## Styling
+
+See the [example story](https://amp.dev/documentation/examples/components/amp-story-interactive-poll/story#page=title-themes) for a demonstration of available amp-story-interactive experiences.
+
+[tip type="read-on"]
+Check this [Codepen collection](https://codepen.io/collection/DEGRLE) to play with the components and styles.
+[/tip]
+
+### CSS Variables
+
+All interactive components support similar styling, which is controlled through CSS variables and attributes.
+CSS variables
+Some variables can be overridden through assigning a class to the component.
+
+- `--interactive-accent-color`: The accent color of the component. If no prompt-background is specified, it will use that for the prompt background as well.
+- `--interactive-prompt-text-color`: Color of the top text. Only used on selectable components if a
+  prompt is specified, or on results components if there are thresholds (which will color the score).
+- `--interactive-prompt-background`: Background of the top text. Only used on selectable components if a prompt is specified, or on results if there are thresholds but no image (which will color the score background). Can be a color (including transparent) or CSS gradient.
+- `--interactive-prompt-alignment`: Alignment of the prompt. Only used in selectable components (not results). Will default to center if the component has the transparent style or if it's a binary poll, otherwise it will default to initial.
+
+### Themes
+
+The `theme` attribute controls the chip color and text color of the component. Can be `light` (default) or `dark`.
+The `chip-style` attribute controls the style details of the component. Can be `flat` (default), `shadow` or `transparent`.
+
+## Sizing
+
+The component follows the [container](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/layouts_demonstrated/#container) model. The size can be changed by overriding the element's font-size, which by default will be set to `3*var(--story-page-vmin)`. The component has a `min-width: 14em` and `max-width: 25em`, unless it's a binary-poll or results, in which case it will have `max-width:18em`. This makes it occupy 75% of the width on portrait stories by default, but the width can be overridden with a CSS rule to any value in between the min and max. The height will depend on the font-size and number of lines on the prompt, so it cannot be specified.
+
+### Creating pixel-perfect layouts using amp-story-grid-layer[aspect-ratio]
+
+While the component by default adapts to the screen size with the variable font-size, it doesn't stay perfectly consistent across screen sizes. It's possible to use the aspect-ratio layer in order to create layouts that will scale perfectly with different screen sizes, by setting the font-size in ems on the component.
```

Comment:
Let's sync on this one offline, I think you wrote this with a ton of context in mind that very very few people have. We should think of how to better communicate all this context.

Reply:
Sure! I rewrote a bit after following the suggestions in previous comments from @CrystalOnScript but will show you